src/v2/guide/instance.md

```diff
@@ -113,39 +113,39 @@ var vm = new Vue({
+Setiap instan Vue melewati serangkaian langkah inisialisasi saat dibuat - misalnya, perlu mengatur observasi data, mengkompilasi template, `mount` instan ke DOM, dan memperbarui DOM ketika data berubah. Sepanjang jalan, itu juga menjalankan fungsi yang disebut **lifecycle hooks**, memberikan pengguna kesempatan untuk menambahkan kode mereka sendiri pada tahap tertentu.
 
-For example, the [`created`](../api/#created) hook can be used to run code after an instance is created:
+Sebagai contoh, [`membuat`](../api/#created) hook dapat digunakan untuk menjalankan kode setelah sebuah instan dibuat:
 
 ```js
 new Vue({
   data: {
     a: 1
   },
   created: function () {
-    // `this` points to the vm instance
+    // `this` mengarahkan ke vm instan
     console.log('a is: ' + this.a)
   }
 })
-// => "a is: 1"
+// => "a adalah: 1"
 ```
 
-There are also other hooks which will be called at different stages of the instance's lifecycle, such as [`mounted`](../api/#mounted), [`updated`](../api/#updated), and [`destroyed`](../api/#destroyed). All lifecycle hooks are called with their `this` context pointing to the Vue instance invoking it.
+Ada juga `hook` lainnya yang akan dipanggil pada berbagai tahap (lifecycle), seperti [`mounted`](../api/#mounted),  [`updated`](../api/#updated), dan [`destroyed`](../api/#destroyed). Semua lifecycle hook dipanggil dengan `this` yang konteksnya menunjuk ke instan Vue yang memintanya.
 
-<p class="tip">Don't use [arrow functions](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) on an options property or callback, such as `created: () => console.log(this.a)` or `vm.$watch('a', newValue => this.myMethod())`. Since arrow functions are bound to the parent context, `this` will not be the Vue instance as you'd expect, often resulting in errors such as `Uncaught TypeError: Cannot read property of undefined` or `Uncaught TypeError: this.myMethod is not a function`.</p>
+<p class="tip">Jangan gunakan [Fungsi Panah (Arrow Function)](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) pada properti opsi atau panggilan balik (callback), seperti `created: () => console.log(this.a)` atau `vm.$watch('a', newValue => this.myMethod())`. Karena fungsi panah terikat pada konteks induk, `this` tidak akan menjadi instan Vue seperti yang kalian harapkan, sering menghasilkan kesalahan seperti `Uncaught TypeError: Cannot read property of undefined` atau `Uncaught TypeError: this.myMethod is not a function`.</p>
 
-## Lifecycle Diagram
+## Diagram Lifecycle
 
-Below is a diagram for the instance lifecycle. You don't need to fully understand everything going on right now, but as you learn and build more, it will be a useful reference.
+Di bawah ini adalah diagram untuk instan Lifecycle. kalian tidak perlu sepenuhnya memahami segala sesuatu yang terjadi saat ini, tetapi saat kalian belajar dan membangun lebih banyak, itu akan menjadi referensi yang sangat berguna.
```

Comment:
Hi @rohmad-st! Thank you for your hard work! I just added a few feedbacks. Let's review it together.